workflows/data_pipelines/finess/config.py

```diff
@@ -9,8 +9,13 @@
     tmp_folder=f"{DataSourceConfig.base_tmp_folder}/finess",
     minio_path="finess",
     file_name="finess",
-    resource_id="2ce43ade-8d2c-4d1d-81da-ca06c82abc68",
-    url=f"{DATA_GOUV_BASE_URL}2ce43ade-8d2c-4d1d-81da-ca06c82abc68",
+    files_to_download={
+        "finess": {
+            "url": f"{DATA_GOUV_BASE_URL}2ce43ade-8d2c-4d1d-81da-ca06c82abc68",
+            "resource_id": "2ce43ade-8d2c-4d1d-81da-ca06c82abc68",
+            "destination": f"{DataSourceConfig.base_tmp_folder}/finess/finess-download.csv",
+        }
+    },
```

Comment:
I think I would just remove the extra layer. What do you think?

Reply:
See in https://github.com/annuaire-entreprises-data-gouv-fr/search-infra/pull/441#discussion_r1895532423